subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -397,6 +429,39 @@ static int prepare_cb_common(struct lll_prepare_param *p)
 	return 0;
 }
 
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_SLOT_WINDOW_JITTER)
+static int resume_prepare_cb(struct lll_prepare_param *p)
+{
+	uint32_t ticks_at_expire;
+	struct ull_hdr *ull;
+
+	ticks_at_expire = p->ticks_at_expire;
+
+	ull = HDR_LLL2ULL(p->param);
+	p->ticks_at_expire = ticker_ticks_diff_get(ticker_ticks_now_get(),
+						   lll_event_offset_get(ull));
+	p->ticks_drift += ticker_ticks_diff_get(p->ticks_at_expire,
+						ticks_at_expire);
+	p->remainder = 0U;
+	p->lazy = 0U;
+
+	return prepare_cb(p);
+}
+#endif /* CONFIG_BT_CTLR_SYNC_ISO_SLOT_WINDOW_JITTER */
+
+static int is_abort_cb(void *next, void *curr, lll_prepare_cb_t *resume_cb)
+{
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_SLOT_WINDOW_JITTER)
+	if (!next) {
+		*resume_cb = resume_prepare_cb;
+
+		return -EAGAIN;
+	}
+#endif /* CONFIG_BT_CTLR_SYNC_ISO_SLOT_WINDOW_JITTER */
+
+	return -ECANCELED;
+}
```

Comment:
Since you don't document internal functions like the `is_abort_cb` it's hard to follow and understand. 

Why are we here returning a pointer to a `resume_cb` while also returning an error, but not in most (all?) other cases?

Reply:
ok... it is not documented...i agree... i should...
0 - pre-emptee succeeds to continue radio use, and pre-emptor is informed
-EAGAIN - pre-emptee looses and requests a resumption later, and pre-emptor will be scheduled
-ECANCELED - pre-emptee looses and be aborted, and pre-emptor will be scheduled